packages/@orchidui-vue/src/composables/helpers.js

```diff
@@ -32,3 +32,29 @@ export const setCookie = (name, value, hours = 9600) => {
     console.log(error);
   }
 };
+
+export const preventEventIfNotNumberInput = (event, options = {}) => {
+  const { isAmountInput } = options;
+
+  const acceptedCharacters = [];
+
+  if (isAmountInput) {
+    acceptedCharacters.push(["."]);
+  }
+
+  if (
+    (event.key >= "0" && event.key <= "9") ||
```

Comment:
We have an issue when entering input using different keyboards, do we need to filter the input from the user, or just display an error message? cc @spacemonkey92 @mykhailo-diakovych